### DISCUSSION TITLE
How to avoid generating a .d.ts when it is already existed? I think it should copy the original .d.ts file

### DISCUSSION THREAD

**Discussion by lz1998** [TypeStrong/ts-node#1199] [_ Off-topic] [answered]
I have a TypeScript project. Most of the code is written in TS, but there is a file generated by protobufjs called `proto.js`. And there is a generated `proto.d.ts`.
When I execute tsc, it doesn't copy the original `proto.d.ts`to `lib` directory, but  generate another `proto.d.ts`.

Project: https://github.com/ProtobufBot/js-pbbot

src/proto/proto.d.ts
```ts
import * as $protobuf from "protobufjs";
/** Namespace onebot. */
export namespace onebot {

    /** Properties of a Frame. */
    interface IFrame {

        /** Frame botId */
        botId?: (Long|null);

......
```

Run `tsc --build tsconfig.json`

lib/proto/proto.d.ts
```ts
export = $root;
declare var $root: $protobuf.Root;
import $protobuf = require("protobufjs/minimal");
```

I think it should copy the original .d.ts file when .d.ts file is existed, but it doesn't.

**Comment by cspotcode** [comment, marked as answer]
This is a `tsc` question, not a `ts-node` question, so I'm going to close it as off-topic.

**Reply by lz1998** [reply]
Which repo should I create this issue?
